### ISSUE TITLE
Python3

### ISSUE THREAD

**Issue by essobi** [encode/jsonhyperschema-codec#10]
Python3 doesn't have urllib.quote, but has urllib.parse.quote.
__init__.py throws an error around line 46 for urllib.unquote.

adding import urllib.parse and changing..
```
-                        item = urllib.unquote(item.strip('(').rstrip(')'))
+                        item = urllib.parse.unquote(item.strip('(').rstrip(')'))
```
seems to work fine from initial testing.

**Comment by lovelydinosaur** [comment]
Okay. Just as a data point, are you using JSON Hyperschema to describe your API?
Trying to get a feel for how important (or not) it is for us to maintain this codec.

**Comment by lovelydinosaur** [comment]
Resolved in 1.0.3
